crates/tools/src/main.rs

```diff
@@ -17,7 +17,10 @@ fn main() -> Result<()> {
         .subcommand(SubCommand::with_name("fuzz-tests"))
         .get_matches();
     match matches.subcommand_name().expect("Subcommand must be specified") {
-        "install-code" => install_code_extension()?,
+        "install-code" => {
+            setup_environment()?;
+            install_code_extension()?;
+        }
```

Comment:
It doesn't matter much, but I would avoid trying to setup a super-generic infrastructure here. Instead, I'd do

```
"install-code" => {
    if cfg!(target_os = "macos") {
       fix_path_for_mac();
    }
    install_code_extension()?;
}

fn fix_path_for_mac() {
    // just a single function which does everything
}
```

Reply:
fixed